fullstory-segment-middleware/src/main/java/com/fullstorydev/fullstory_segment_middleware/FSSuffixedProperties.java

```diff
@@ -0,0 +1,141 @@
+package com.fullstorydev.fullstory_segment_middleware;
+
+import androidx.annotation.VisibleForTesting;
```

Comment:
just so you know, this creates a a dependency on the androidx library and will prevent anybody that still uses the old support libraries from using this.

Reply:
Thanks for the catch! Since the methods are all `protected` and the tests can get to it, I will remove this.